tests/ui/consts/min_const_fn/recursive_const_stab_unstable_if_unmarked.rs

```diff
@@ -1,6 +1,6 @@
 //@ compile-flags: -Zforce-unstable-if-unmarked
 //@ edition: 2021
-#![feature(const_async_blocks, rustc_attrs, rustc_private)]
+#![feature(const_async_blocks, rustc_attrs)]
```

Comment:
Without this PR, removing `rustc_private` here would have completely changed the output. So the point of the PR is that even without `rustc_private`, the output here is as before: suggesting `rustc_const_stable_indirect` on the callee.